src/board.c

```diff
@@ -3,19 +3,29 @@
 #include <assert.h> // Dla assert
 #include <stdio.h> // Dla printf
 
+void board_content_assert(int** arrayToCheck, size_t rows, size_t cols)
+{
+    for(size_t j = 0; j < rows; j++){
+        for(size_t i = 0; i < cols; i++){
+            assert((arrayToCheck[j][i] >= 1 && arrayToCheck[j][i] <= 8)||(arrayToCheck[j][i] == 0 || arrayToCheck[j][i] <= -1 || arrayToCheck[j][i] >= -3 )); // Sprawdzamy czy wartość pola jest poprawna
```

Comment:
ten assert dla np. arrayToCheck[j][I] = 420682137 zwróci wartość poprawna bo jest >= -3 i operator || (or) itp. Jeśli chcemy tu sprawdzać czy pola zawierają poprawne indeksy to trzeba to na sztywno przypisać tak jak wcześniej było.

Reply:
prawda, ale lepiej chyba będzie po prostu zamienić niektóre operatory || na &&, sprawdzać na sztywno ponad 10 liczb to zaraz będzie wyglądało jak funkcja isEven z memów, dobra uwaga, siadam do tego zaraz

Reply:
@oskar-przybylski już naprawione - commit 9ec80e9c3d8afb5ad837b3c4640ea44ca4365ba5